packages/launchpad/src/main.scss

```diff
@@ -1,6 +1,7 @@
 /* Define the "system" font family */
 @font-face {
   font-family: system;
+  // font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
Can delete this

Reply:
Oh uh. No, I need to revert that. We want the larger fontstack.